Uno.Gallery/Uno.Gallery.UITest/Given_Card_01.cs

```diff
@@ -0,0 +1,150 @@
+			var elevatedCardClick = new QueryEx(x => x.All().Marked("Elevated_Card"));
+			App.ScrollDownTo("Elevated_Card");
+			elevatedCardClick.Tap();
+			TakeScreenshot("After Click");
+			Assert.IsTrue(elevatedCardClick.GetDependencyPropertyValue<bool>("IsEnabled"));
+			Assert.AreEqual("Elevated card", elevatedCardClick.GetDependencyPropertyValue("HeaderContent"));
+			Assert.AreEqual("With supporting content", elevatedCardClick.GetDependencyPropertyValue("SubHeaderContent"));
+			Assert.AreEqual("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.", elevatedCardClick.GetDependencyPropertyValue("SupportingContent"));
+		}
+
+		/* This function is to test the Checked option in checkbox for material.	*/
+		[Test]
+		public void When_03_OutlinedCardWithMediaContentClick()
+		{
+			NavigateToSample("Card", "Material");
+
+			TakeScreenshot("Before Click");
+			var outlinedCardMediaContent = new QueryEx(x => x.All().Marked("Outlined_Card_Media_Content"));
+			App.ScrollDownTo("Outlined_Card_Media_Content");
+			outlinedCardMediaContent.Tap();
+			TakeScreenshot("After Click");
+			Assert.IsTrue(outlinedCardMediaContent.GetDependencyPropertyValue<bool>("IsEnabled"));
+			Assert.AreEqual("Outlined card", outlinedCardMediaContent.GetDependencyPropertyValue("HeaderContent"));
+			Assert.AreEqual("With media content", outlinedCardMediaContent.GetDependencyPropertyValue("SubHeaderContent"));
+			Assert.AreEqual("ms-appx:///Assets/LargeMedia.png", outlinedCardMediaContent.GetDependencyPropertyValue("MediaContent"));
+		}
+
+		/* This function is to test the DisabledChecked option  in checkbox for material */
+		[Test]
+		public void When_04_ElevatedCardWithMediaContentClick()
+		{
+			NavigateToSample("Card", "Material");
+
+			TakeScreenshot("Before Click");
+			var elevatedCardMediaContent = new QueryEx(x => x.All().Marked("Elevated_Card_Media_Content"));
+			App.ScrollDownTo("Elevated_Card_Media_Content");
+			elevatedCardMediaContent.Tap();
+			TakeScreenshot("After Click");
+			Assert.IsTrue(elevatedCardMediaContent.GetDependencyPropertyValue<bool>("IsEnabled"));
+			Assert.AreEqual("Elevated card", elevatedCardMediaContent.GetDependencyPropertyValue("HeaderContent"));
+			Assert.AreEqual("With media and supporting content", elevatedCardMediaContent.GetDependencyPropertyValue("SubHeaderContent"));
+			Assert.AreEqual("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.", elevatedCardMediaContent.GetDependencyPropertyValue("SupportingContent"));
+			Assert.AreEqual("ms-appx:///Assets/LargeMedia.png", elevatedCardMediaContent.GetDependencyPropertyValue("MediaContent"));
+		}
+
+		/* This function is to test the Indeterminate for uncheck, recheck and indeterminate option in checkbox for material   */
+		[Test]
+		public void When_05_OutlinedCardWithMediaSupportingContent_SupplementalActionButtonClick()
+		{
+			NavigateToSample("Card", "Material");
+
+			TakeScreenshot("Before Click");
```

Comment:
If you can add a little more comments for the tests like you did on the chip PR please ;)